client/src/pages/Dashboard/components/ModalReact.js

```diff
@@ -16,15 +18,18 @@ function ModalReact({ setOpenModal, intentName, intentFrequency, intentAssociate
 
       <Modal.Footer>
         <Button variant="secondary" onClick={() => {
-              setOpenModal(false);
-            }}>Close</Button>
+          setOpenModal(false);
+        }}>Close</Button>
         <Button variant="primary" onClick={() => {
-              // DIRECT US TO VOICEFLOW HERE
-              // use {intentName}
-            }}>Add Intent to Voiceflow</Button>
+          // DIRECT US TO VOICEFLOW HERE
```

Comment:
No need for the comment, lets remove it.

Reply:
True, it doesn't accurately reflect what we're doing in this function anyway. Removed it!